monkey/infection_monkey/monkey.py

```diff
@@ -32,11 +32,17 @@
 from common.utils.attack_utils import ScanStatus, UsageEnum
 from infection_monkey.exploit.HostExploiter import HostExploiter
 
+MAX_DEPTH_REACHED_MESSAGE = "Reached max depth, shutting down"
+
 __author__ = 'itamar'
 
 LOG = logging.getLogger(__name__)
 
 
+class PlannedShutdown(Exception):
```

Comment:
Would be better if it was named `PlannedShutdownException`

Reply:
a3d81a00